docker-compose.yml

```diff
@@ -44,6 +44,10 @@ services:
       nofile:
         soft: 65536
         hard: 65536
+    # Memory limit for ES, as it tends to be a memory hoarder
+    # Set this value to an empty string to remove the limit
+    # https://docs.docker.com/compose/compose-file/compose-file-v2/#cpu-and-other-resources
+    mem_limit: ${ES_MEM_LIMIT:-4g}
```

Comment:
That's entirely possible! If you're on a Mac, you probably have. I'm running this on Ubuntu, and there seems to be no system-level limits by default. I'll look into changing this! Either way though, I think this is a good change to prevent other contributors from having all their memory consumed :sweat_smile: